parse_emails/handle_eml.py

```diff
@@ -273,6 +275,34 @@ def handle_eml(file_path, b64=False, file_name=None, parse_only_headers=False, m
         return email_data, attached_emails
 
 
+def create_message_from_string(file_data):
```

Comment:
```suggestion
def create_message_from_string(file_data: str) -> Message:
```
Let's also add a short docstring